### PR TITLE
Merge into master

### PR DESCRIPTION
**Pull request by MohamedAbdelsalam9**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## IIRC: Incremental Implicitly Refined Classification
```
Comment (bemova) [inline README.md]: please add a citation section. like https://github.com/chandar-lab/patchup#citation

Comment (bemova) [reply README.md]: it will be also nice to put the web page link in the future.

File: create_confusion.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
+import torch
```
Comment (bemova) [inline create_confusion.py]: what does the file name meaning? can we put a relevant name?

**Review by bemova — COMMENTED** [review]
just a few high-level comments.

File: create_confusion.py (hunk 2 above)
Comment (MohamedAbdelsalam9) [reply create_confusion.py]: it's confusion matrix, will rename it to create_confusion_mat

File: README.md (hunk 1 above)
Comment (MohamedAbdelsalam9) [reply README.md]: I will update the whole readme

**Pull request closed without merging**